blog/_posts/2023-05-20-sbt-community-repository-follow-up.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Scala Center
+title: "sbt Community Repository (Follow-Up)"
+---
+
+[Last month]({% link blog/_posts/2023-04-20-sbt-plugins-community-repository.md %}), I published a blog article where I described the sbt community repository. In this follow-up article, I present our plan to strengthen the infrastructure that supports the sbt ecosystem.
+
+This blog article is also an initial response to the related Advisory Board proposal [SCP-029 sbt community repository](https://github.com/scalacenter/advisoryboard/blob/main/proposals/029-sbt-community-repository.md), which has been approved by the Advisory Board during the meeting of the [27th of April](https://github.com/scala/scala.epfl.ch/blob/main/minutes/_posts/2023-04-27-april-27-2023.md).
+
+## Current State
+
+The sbt community repository was set up a couple of years ago with two primary purposes:
+1. host “old” versions of sbt plugins so that “old” Scala projects continue to work today and in the future,
```

Comment:
```suggestion
1. To host “old” versions of sbt plugins so that “old” Scala projects continue to work today and in the future
```

Reply:
also both uses of scare quotes seem strange to me. I would suggest removing the quotes, as it isn't clear to me what meaning they're intended to convey. if there's a meaning that needs to be conveyed, it would be better to find actual words